app/config/collections/projects.php

```diff
@@ -797,6 +797,364 @@
+                'signed' => true,
+                'required' => false,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => ID::custom('deploymentId'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => Database::LENGTH_KEY,
+                'signed' => true,
+                'required' => false,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => ID::custom('vars'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 16384,
+                'signed' => true,
+                'required' => false,
+                'default' => null,
+                'array' => false,
+                'filters' => ['subQueryVariables'],
+            ],
+            [
+                '$id' => ID::custom('varsProject'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 16384,
+                'signed' => true,
+                'required' => false,
+                'default' => null,
+                'array' => false,
+                'filters' => ['subQueryProjectVariables'],
+            ],
+            [
+                '$id' => ID::custom('timeout'),
+                'type' => Database::VAR_INTEGER,
+                'format' => '',
+                'size' => 0,
+                'signed' => true,
+                'required' => false,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => ID::custom('search'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 16384,
+                'signed' => true,
+                'required' => false,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                'array' => false,
+                '$id' => ID::custom('specification'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 128,
+                'signed' => false,
+                'required' => false,
+                'default' => APP_COMPUTE_SPECIFICATION_DEFAULT,
+                'filters' => [],
+            ],
+            [
+                '$id' => ID::custom('buildRuntime'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 2048,
+                'signed' => true,
+                'required' => true,
+                'default' => '',
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => ID::custom('adapter'), // ssr or static
```

Comment:
is adapter really the best name here? Should we consider `rendering`?

Reply:
A framework can support multiple of adapter, for example SSR and static.

Due to that, we need a key for it `frameworks[0].adapters[0].buildCommand`.

Notice we dont use `adapter` there, but `adapters`, since it's array.

I couldn't find a word better than adapter that would have good plural.

`renderings` sounds bad, more technically correct is `strategies`. But then `site.adapter` sounds much better than `site.strategy`. Strategy in this sense (to me) sounds more like deployment term - strategy to decide when to activate deployment on production

For those reasons I prefer adapter. If you have more suggestions, open to discuss